polkadot/xcm/src/v3/junction.rs

```diff
@@ -437,6 +437,14 @@ impl Junction {
 			_ => {},
 		}
 	}
+
+	/// Specifies whether junction identifies a chain.
+	pub fn is_chain_identifier(&self) -> bool {
```

Comment:
I don't feel like this is useful as a method on `Junction`s. The function body can be replaced by a `matches!` macro most of the time.

Reply:
done